test/sqlite/speedtest1.c

```diff
@@ -18,7 +18,7 @@ static const char zHelp[] =
   "  --exclusive         Enable locking_mode=EXCLUSIVE\n"
   "  --explain           Like --sqlonly but with added EXPLAIN keywords\n"
   "  --heap SZ MIN       Memory allocator uses SZ bytes & min allocation MIN\n"
-  "  --incrvacuum        Enable incremenatal vacuum mode\n"
+  "  --incrvacuum        Enable incremental vacuum mode\n"
```

Comment:
I imagine this code is form upstream sqliite?

Reply:
The file has the Emscripten license so fixed it.. But searching that specific typo from Github results in thousands of lines: https://github.com/search?q=%22Enable+incremenatal+vacuum+mode%22&type=code. Reverted